docs/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Sphinx>=3.0
+sphinx-rtd-theme>=0.5.2
```

Comment:
I assume this is needed to build the documentation locally. 2 thoughts:

- Maybe the doc-related changes are worth a separate pull request.
- Maybe we should take the chance to provide instructions to build the documentation locally somewhere in the README, and mention the installation of these requirements.